include/warehouse_ros/transform_collection.h

```diff
@@ -41,7 +41,7 @@
 #include <tf2_ros/transform_listener.h>
 #include <tf2_msgs/msg/tf_message.hpp>
 #include <geometry_msgs/msg/transform_stamped.hpp>
-#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
```

Comment:
```suggestion
#if __has_include(<tf2_geometry_msgs/tf2_geometry_msgs.hpp>)
#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
#else
#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
#endif```

Reply:
Since galactic does not have these headers, we decided to use preprocessor directives to handle that https://en.cppreference.com/w/cpp/preprocessor/include, can you please change accordingly and we can merge @Drojas251

Reply:
@Drojas251 any updates on this?